sdks/python/apache_beam/io/parquetio_test.py

```diff
@@ -109,6 +111,37 @@ def setUp(self):
                                ('favorite_number', pa.timestamp('ns'), False),
                                ('favorite_color', pa.string())])
 
+    self.RECORDS_NESTED = [{
+        'items': [
+            {
+                'name': 'Thomas',
+                'favorite_number': 1,
+                'favorite_color': 'blue'
+            },
+            {
+                'name': 'Henry',
+                'favorite_number': 3,
+                'favorite_color': 'green'
+            },
+        ]
+    },
+                           {
```

Comment:
Is this a correct indentation? Shouldn't this be the same level with the previous `items`?

Reply:
Yes, it is correct from PythonFormatter.

If I keep the it with same level with the previous item, the PythonFormatter PreCommit fails.